frontend/src/Component/Tables/TableSubComponents/Admin/AdminTable.jsx

```diff
@@ -7,27 +8,39 @@ import useSortableData from "../../../../Hooks/Tables/useSortableTable";
 const apiKey = import.meta.env.VITE_BACKEND_API_KEY;
 
 const AdminTable = () => {
-    const { devices: initialData, apiError } = useGetDeviceList(`${apiKey}/api/v1/devices`);
-    const { sortedData, onSort, sortConfig } = useSortableData(initialData);
-    
-    return (
-        <>
-        {sortedData.length === 0 ? (
-          <h1 className={'text-2xl text-center'}>No data available</h1>
-        ) : (
-          <>
-          {apiError ? (
-            <div className={'bg-red-500 text-white p-4'}>Error: {apiError}</div>
-          ) : (
-            <table className={'w-full text-sm text-left text-gray-500 divide-y divide-gray-200'}>
-              <TableHeaders headers={tableHeaders} onSort={onSort} sortConfig={sortConfig} />
-              <AdminTableBody tableFields={sortedData} />
-            </table>
-          )}
-          </>
-          )}
-        </>
-        );
-    };
-    
-    export default AdminTable;
\ No newline at end of file
+  const { devices: initialData, apiError } = useGetDeviceList(`${apiKey}/api/v1/devices`);
+  const { sortedData, onSort, sortConfig } = useSortableData(initialData);
```

Comment:
The table no longer users the sortedData function, and the sorting no longer works.